rxandroid/src/main/java/rx/android/lifecycle/LifecycleHelper.java

```diff
@@ -0,0 +1,92 @@
+package rx.android.lifecycle;
+
+import android.app.Activity;
+import android.app.Application;
+import android.os.Bundle;
+
+import rx.subjects.BehaviorSubject;
+
+/**
+ * This instance lives within the scope of an {@code Activity} in order to forward Lifecycle events
+ * to {@link rx.subjects.BehaviorSubject}
+ *
+ * Thread safe, since it's <i>Thread-confined</i> to the main Thread.
+ *
+ * @author Nabil Hachicha.
```

Comment:
Remove. This library is free of individual authors and individual copyright owners.